names.cpp

```diff
@@ -8,168 +8,168 @@
-            << "Ultimecia's Castle"
-            << "Ultimecia's Castle"
-            << "Ultimecia's Castle"
-            << "Eshtar City"
-            << "Lunatic Pandora Lab"
-            << "Lunatic Pandora Lab"
-            << "Edea's Parade Vehicle"
-            << "Tomb of the Unknown King (Boss Fight room)?"
-            << "Desert Prison?"
-            << "Galbadian something?"
-            << "Generic Landscape?"
-            << "Generic Landscape?"
-            << "Balamb Garden (External Corridor?)"
-            << "Balamb Garden (External Corridor?)"
-            << "Balamb Garden (External Corridor?)"
-            << "Balamb Garden (External Corridor?)"
-            << "Balamb Garden (External Corridor?)"
-            << "Generic Landscape?"
-            << "Generic Landscape?"
-            << "Generic Landscape?"
-            << "Test Environment? (UV tile texture)"
-            << "Generic Landscape?"
-            << "Generic Landscape?";
+			<< "Dollet Bridge"
+			<< "Dollet Trasmission Tower path"
+			<< "Dollet Transmission Tower (Top)"
+			<< "Dollet Transmission Tower (Elevator)"
+			<< "Dollet Transmission Tower (Elevator [UNUSED?])"
+			<< "Dollet City"
+			<< "Balamb Garden entrance gate"
+			<< "Balamb Garden 1st Floor Hall"
+			<< "Balamb Garden 2nd Floor Corridor"
+			<< "Balamb Garden Quad [Balamb raid version]"
+			<< "Balamb Garden Outer Corridor"
+			<< "Balamb Garden Training Center (elevator zone)"
+			<< "Balamb Garden Norg's Floor"
+			<< "Balamb Garden Underground Levels (Tube)"
+			<< "Balamb Garden Underground levels"
+			<< "Balamb Garden Underground levels [OilBoil]"
+			<< "Timber Pub Area"
+			<< "Timber Maniacs square"
+			<< "Train (Deling Presidential Vagon)"
+			<< "Deling City Sewers"
+			<< "Deling City (Caraway Residence secret exit)"
+			<< "Balamb Garden Class Room"
+			<< "Galbadia Garden Corridor"
+			<< "Galbadia Garden Corridor 2"
+			<< "Galbadia Missile Base"
+			<< "Deep Sea Research Center [Bahamut]"
+			<< "Balamb Town (Balamb Hotel road)"
+			<< "Balamb Town (Balamb Hotel Hall)"
+			<< "[UNUSED?]"
+			<< "Fire Cavern (path)"
+			<< "Fire Cavern (Ifrit Lair)"
+			<< "Galbadia Garden Hall"
+			<< "Galbadia Garden Auditorium (Seifer battle CD2)"
+			<< "Galbadia Garden Auditorium 2 [Seifer + Edea CD2]"
+			<< "Galbadia Garden Corridor"
+			<< "Galbadia Garden (Ice Hockey Field)"
+			<< "Ultimecia - griever"
+			<< "Ultimecia - last form"
+			<< "Desert Prison [UNUSED?]"
+			<< "Desert Prison"
+			<< "Eshtar City (road)"
+			<< "Desert Prison [Top]"
+			<< "Eshtar City (road2)"
+			<< "Missile Base Inside [UNUSED?]"
+			<< "Missile Base? Inside [UNUSED?]"
+			<< "Missile Base Control room"
+			<< "Winhill Village main square"
+			<< "Tomb of the Unknown King (Corridor)"
+			<< "Eshtar City (road 3)"
+			<< "Tomb of the Unknown King (Boss Fight)"
+			<< "Fisherman Horizon (Road)"
+			<< "Fisherman Horizon (Train Station Square)"
+			<< "Desert Prison Cell (UNUSED?)"
```

Comment:
There are encounters in the Squall prison cell, but you cannot trigger it since you can't go back in there after the scenes with Seifer.